packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -34,7 +34,7 @@ async function runPipelineLoadStep(
   srcPath: string,
   {buildPipeline, messageBus, isDev}: BuildFileOptions,
 ) {
-  const srcExt = getExt(srcPath).baseExt;
+  const srcExt = getExt(srcPath).expandedExt;
```

Comment:
So interesting question on extensions. Consider a file like `.module.css`: 

- `baseExt`: '.css'
- `expandedExt`: '.module.css'

We only pass one to the plugin: `fileExt`. If you made a plugin, which would you expect to match? In some cases you only want the base; in others you want the full extension.

Reply:
yea I'd been thinking through this as well: we may want to revisit `expandedExt` as a concept. I'm uneasy with how it behaves if the user uses `.`'s in their file names, the way we saw users report that they were using `foo.factory.js` and `foo.service.js`. 

The idea that `expandedExt === '.module.css'` isn't always true makes me think that more advanced checks for things like `.module.css` and `expandedExt` should be handled with `.endsWith()`, at least as far as our plugin interface goes. We can always add later if we get this request often, but I'd rather not add it before then given my uncertainty.

Reply:
Makes sense. I’ll revert back to `baseExt` before merging then. But I think we should put more thought into by default making plugins “less matchy,” as the `.css` vs `.module.css` being really exclusive extensions with no overlap has been a royal pain in my opinion (if you’re using CSS Modules you probably don’t want overlap between the two).

Reply:
Yup that's the tricky bit, sometimes you do want to match both and sometimes you don't.